src/Api/Api.js

```diff
@@ -0,0 +1,3 @@
+const ROCKETS_API = 'https://api.spacexdata.com/v3/rockets';
+// const MISSIONS_API = 'https://api.spacexdata.com/v3/missions';
```

Comment:
Please kindly remove unused code from your code base. it will make your code clean.